src/DGtal/geometry/curves/estimation/SegmentComputerEstimators.h

```diff
@@ -1239,12 +1238,12 @@ namespace DGtal
       template<typename DSS>
       Value operator() (const DSS& aDSS) const 
       {
-	typedef typename DSS::Vector Vector; 
-	//length
-	Vector v = ( *aDSS.begin() - *boost::prior(aDSS.end()) ); 
-	Value l = v.norm(Vector::L_2);
-	//result
-	return 1/( (l*l)/8 + 0.5 );  
+        typedef typename DSS::Vector Vector; 
+        //length
+        Vector v = ( *aDSS.begin() - *boost::prior(aDSS.end()) ); 
+        Value l = v.norm(Vector::L_2);
+        //result
+        return 1/( (l*l)/8 + 0.5 );  
```

Comment:
Is `l.norm()` always bigger than `0`? Also, maybe 1 -> 1. and 8 -> 8.

Reply:
I'm not sure, I just fixed indentation.
But I think we can only have 0 if we deal with a DSS a size 0, i.e. than the first element of `aDSS` is the same than the last.

Reply:
done